pkg/dhclient/dhclient.go

```diff
@@ -132,6 +162,16 @@ type Config struct {
 	// Retries is how many times to retry DHCP attempts.
 	Retries int
 
+	// LinkUpTimeout is the time to try setting each link up.
+	//
+	// The default is 30 seconds if none is set.
+	LinkUpTimeout time.Duration
```

Comment:
just wondering, is there any doc anywhere that says 30 seconds is a good idea?

Reply:
It depends on the network and your network driver. In most cases it should come up quickly, but when it doesn't, drivers generally don't have a great way to report what is going wrong.